.github/workflows/config/spelling_allowlist.txt

```diff
@@ -91,9 +95,11 @@ entanglers
 enum
 et
 executables
+https
```

Comment:
Same here. The spell check should not have an issue with links in the docs. If something is not getting filtered, we will adapt the config.